test/acceptance/top-contributors-test.js

```diff
@@ -74,6 +74,13 @@ acceptance("Right Sidebar - Top Contributors - Custom count", function (needs) {
   test("Viewing latest", async function (assert) {
     await visit("/");
 
+    assert
+      .dom(".order--likes_given")
+      .isNotVisible("likes_given class is not added");
+    assert
+      .dom(".order--likes_received")
+      .isVisible("likes_received class is added correctly");
```

Comment:
To test exactly what messages suggested 😃 

```suggestion
    assert
      .dom(".top-contributors--user-likes")
      .doesNotHaveClass(".order--likes_given")
    assert
      .dom(".top-contributors--user-likes")
      .doesHaveClass(".order--likes_received")
```